frontend/app/services/TouchpointBackend.scala

```diff
@@ -60,10 +65,9 @@ object TouchpointBackend {
 
 case class TouchpointBackend(memberRepository: FrontendMemberRepository,
                              stripeService: StripeService,
-                             zuoraSoapService: ZuoraSoapService,
-                             zuoraRestService: ZuoraRestService) extends ActivityTracking {
-
-  val subscriptionService = new SubscriptionService(zuoraSoapService, zuoraRestService)
+			     zuoraSoapClient: soap.ClientWithFeatureSupplier,
+			     zuoraRestClient: rest.Client,
```

Comment:
Yep, this isn't a blocker to this PR going out. I think overall I prefer the package-based naming, in any case.